SPECS-EXTENDED/tardev-snapshotter/tardev-snapshotter.spec

```diff
@@ -0,0 +1,77 @@
+%global debug_package %{nil}
+
+Summary: Tardev Snapshotter for containerd
+Name: tardev-snapshotter
+Version: 0.0.13
+Release: 2%{?dist}
+License: ASL 2.0
+Group: Tools/Container
+# URL: https://www.containerd.io
+Vendor: Microsoft Corporation
+Distribution: Azure Linux
+
+# Source0: https://github.com/microsoft/kata-containers/tree/jiria/solar
```

Comment:
it looks like the sourcing is inconsistent? seems to be sourced by a feature branch? or as specified in the manifest by an old 3.2.0.azl2.tar.gz release? Likely this contains more CVEs!?

Reply:
Agree on the sourcing questions.
Regarding the CVE comment, this package is going into SPECS-EXTENDED for highly experimental non-production usage. There should be no expectation of CVE support for these extended packages. (but also, I do agree we should avoid introducing packages with CVEs if avoidable)

Reply:
I think we should fix this comment, as the source is now from a different branch anyway. Also, I have a script in steamboat to automatically recreate the vendor tarball and source tarball. Might be worth to carry that over.

Reply:
I'm not sure this is the right place to maintain a script to generate sources. For example, the kernel spec leaves steps to update config sources in a comment, but I'm also not too opinionated since it is going into specs-extended for now.

@jiria is it enough to leave a comment in the spec with steps to generate sources or do we really need the file?

Reply:
Source comment removed, version is now using a tag, vendor file has been added